File: MovieCatalogQw/app/src/test/java/com/unlistedi/moviecatalogqw/tvshows/TvShowsViewModelTest.java
package com.unlistedi.moviecatalogqw.tvshows;

import com.unlistedi.moviecatalogqw.data.MovieEntity;
import com.unlistedi.moviecatalogqw.ui.tvshows.TvShowsViewModel;

import org.junit.Before;
import org.junit.Test;

import java.util.ArrayList;

import static org.junit.Assert.*;

public class TvShowsViewModelTest {
    private TvShowsViewModel tvShowsViewModel;

    @Before
    public void setUp(){
        tvShowsViewModel = new TvShowsViewModel();
    }

    @Test
    public void getTvShows(){
        ArrayList<MovieEntity> tvShows = tvShowsViewModel.getTvShows();
        assertNotNull(tvShows);
        assertEquals(20, tvShows.size());
    }
}

File: MovieCatalogueKu/app/src/test/java/com/unlistedi/moviecatalogueku/ui/tvshows/TvShowsViewModelTest.kt
package com.unlistedi.moviecatalogueku.ui.tvshows

import org.junit.Assert.*
import org.junit.Before
import org.junit.Test

class TvShowsViewModelTest {
    private lateinit var tvShowsViewModel: TvShowsViewModel

    @Before
    fun setUp(){
        tvShowsViewModel = TvShowsViewModel()
    }

    @Test
    fun getMovies(){
        val tvShows = tvShowsViewModel.tvShows
        assertNotNull(tvShows)
        assertEquals(20, tvShows.size)
    }
}